### PR TITLE
Update for basic-cli 0.13.0

### PR DESCRIPTION
**Pull request by lukewilliamboswell**

### CODE REVIEW

File: rtl.roc [hunk 1]
```diff
@@ -13,19 +14,19 @@ import cli.Utc exposing [Utc]
...
 
 main =
     start = Utc.now!
 
     cliParser =
-        Cli.build {
-            maybeInputDir: <- Opt.maybeStr { short: "i", long: "input-directory", help: "The directory containing the templates to be compiled. Defaults to the current directory." },
-            maybeOutputDir: <- Opt.maybeStr { short: "o", long: "output-directory", help: "The directory Pages.roc will be written to. Defaults to the current directory." },
-            maybeExtension: <- Opt.maybeStr { short: "e", long: "extension", help: "The extension of the template files the CLI will search for. Defaults to `rtl`." },
+        { Arg.Cli.combine <-
```
Comment (isaacvando) [inline rtl.roc:24]: Can this (and the others) be `Cli.combine` instead of `Arg.Cli.combine`?

Comment (lukewilliamboswell) [reply rtl.roc:24]: You prefer the alias? no problem.

Comment (isaacvando) [reply rtl.roc:24]: I do, but on second thought it doesn't really matter either way.

**Comment by isaacvando** [comment]
Closing as this PR has now been superseded by another

**Pull request closed without merging**